common/component/kafka/kafka.go

```diff
@@ -143,8 +143,8 @@ func (k *Kafka) Init(ctx context.Context, metadata map[string]string) error {
 	config.Consumer.Fetch.Default = meta.consumerFetchDefault
 	config.ChannelBufferSize = meta.channelBufferSize
 
-	config.Net.KeepAlive = meta.clientConnectionKeepAliveInterval
-	config.Metadata.RefreshFrequency = meta.clientConnectionRefreshInterval
+	config.Net.KeepAlive = meta.ClientConnectionKeepAliveInterval
+	config.Metadata.RefreshFrequency = meta.ClientConnectionRefreshInterval
```

Comment:
Should this be named something like `meta.ClientMetadataRefreshFrequency` instead to be clear on its purpose?

Reply:
I renamed to ClientConnectionTopicMetadataRefreshFrequency